### ISSUE TITLE
[Accessibility]The contrast between foreground and background colors doesn't meet contrast ratio thresholds

### ISSUE THREAD

**Issue by plabanTest1** [gonfunko/scratch-blocks#144]
Accessibility tools used: Lighthouse & axe DevTools

Ensure the contrast between foreground and background colors meets the minimum contrast ratio thresholds.

Below are the elements with the contrast issue.
- project-title-input_title-field_en5Gd
- menu-bar_account-nav-menu_3uu9p > span
- #react-tabs-2 > span
- #react-tabs-4 > span

**Comment by gonfunko** [comment]
These are all part of Scratch's UI (not the blocks editor) and out of scope for this effort.
